dis_snek/models/discord_objects/guild.py

```diff
@@ -1017,6 +1017,10 @@ async def get_guild_integrations(self) -> List["GuildIntegration"]:
         data = await self._client.http.get_guild_integrations(self.id)
         return [GuildIntegration.from_dict(d | {"guild_id": self.id}, self._client) for d in data]
 
+    async def search_guild_members(self, name: str, limit: int = 1) -> List["Member"]:
```

Comment:
After some thought here it shouldn't be "name" but something else, since it can be any string that could be used to find user nickname or username.

Reply:
Query?

Reply:
How about search_query, for even more clarification? (at least imo)

Reply:
Seems kinda redundant seeing as Query means to ask for information, and the method is literally called search